notebooks/entities-synonyms/theasaurus.py

```diff
@@ -3,6 +3,8 @@
 
```

Comment:
Feel a bit bad about this, but the file name has a typo, should be `thesaurus.py` (I had to double check the correct spelling myself!)

Reply:
Good spot, I didn't know! Changed now in https://github.com/alphagov/govuk-entity-personalisation/pull/5/commits/e11cec671aa29a3097db56ed3856275cb128644b.